modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -137,21 +137,23 @@ spec:
     name: <cluster-image-set> <1>
   networking:
     clusterNetwork:
-    - cidr: 10.128.0.0/14
+    - cidr: <cluster-network-cidr> <2>
       hostPrefix: 23
     machineNetwork:
-    - cidr: <machine-network-cidr> <2>
+    - cidr: <machine-network-cidr> <3>
     serviceNetwork:
-    - 172.30.0.0/16
+    - <service-network-cidr> <4>
   provisionRequirements:
     controlPlaneAgents: 1
     workerAgents: 0
-  sshPublicKey: <public-key> <3>
+  sshPublicKey: <public-key> <5>
 ----
 +
 <1> `cluster-image-set` is the name of the ClusterImageSet custom resource used to install {product-title} on the DU.
-<2> `machine-network-cidr` is the target bare metal machine’s CIDR.
-<3> `public-key` entered as plain text can be used to SSH into the target bare metal machine after the host is installed.
```

Comment:
I'd appreciate some clarity here; the procedure talks about "bare metal server/machine" but the tables were more concerned about "DU" (distributed units?). So I'm not sure if what I have here is accurate anymore. 

Not very sure either if the info about the cluster network cidr is accurate.

Reply:
So the DU is the high level term for the computing infrastructure that processes the 5G network traffic at the network edge. The DU application itself runs on customized bare metal servers deployed in OpenShift. When we are talking in the abstract about configuring hardware, I think it makes sense to refer to bare metal, if we are talking about high level 5G networks, DU works.

Reply:
Agreed. "Machine" or "Node" would be fine. Once it's running and you're sshing into it, being bare metal or not isn't too relevant.